style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
```

Comment:
- [ ] Kindly make sure you have a class to style your element as required in the project description as it is helpful to do so when you have multiple same elements on your page.
Hint: `You may consider using header as class name or the one that will suits best for you` :smile: